deployment/README.md

```diff
@@ -159,28 +161,59 @@ foo@bar:~$ aws emr add-steps \
 
 > **Note**: [Azure](https://azure.microsoft.com/en-us/services/databricks/) and [AWS](https://databricks.com/aws) Databricks is Linux-based. Therefore, if you are interested in deploying your app to Databricks, make sure your app is .NET Standard compatible and that you use [.NET Core compiler](https://dotnet.microsoft.com/download) to compile your app.
 
+Databricks allows you to submit Spark .NET apps to an existing active cluster or create a new cluster everytime you launch a job. This requires the **Microsoft.Spark.Worker** to be installed **first** before you submit a Spark .NET app.
+
 ### Deploy Microsoft.Spark.Worker
 *Note that this step is required only once*
 
-#### Cluster Node Initialization Scripts
-Using Databrick's [init script](https://docs.databricks.com/user-guide/clusters/init-scripts.html) mechanism, we will run a shell script during startup for each cluster node before the Spark driver or worker JVM starts.
+  1. Download **[db-init.sh](../deployment/db-init.sh)** and **[install-worker.sh](../deployment/install-worker.sh)** onto your local machine
+  2. Modify **db-init.sh** appropriately to point to the Microsoft.Spark.Worker release you want to download and install on your cluster
+  3. Download and install [Databricks CLI](https://docs.databricks.com/user-guide/dev-tools/databricks-cli.html)
+  4. [Setup authentication](https://docs.databricks.com/user-guide/dev-tools/databricks-cli.html#set-up-authentication) details for the Databricks CLI appropriately
+  5. Upload the files you downloaded and modified to your Databricks cluster
+     ```
+     cd <path-to-init-db-and-install-worker>
+     databricks fs cp init-db.sh dbfs:/spark-dotnet/
+     databricks fs cp install-worker.sh dbfs:/spark-dotnet/
+     ```
+  6. Goto to your Databricks cluster homepage -> Clusters (on the left-side menu) -> Create Cluster
+  7. After configuring the cluster appropriately, set the init script (see the image below) and create the cluster.
+     
+     <img src="../docs/img/deployment-databricks-init-script.PNG" alt="ScriptActionImage" width="500"/>
```

Comment:
this image looks blurred, try different width?

Reply:
Done